.github/workflows/inspektor-gadget.yml

```diff
@@ -600,6 +601,11 @@ jobs:
         minikube version: 'v1.25.2'
         kubernetes version: 'v1.23.0'
         github token: ${{ secrets.GITHUB_TOKEN }}
+        driver: ${{ matrix.driver }}
+        # When using docker driver, run the integration tests on two nodes.
+        # As GitHub Runner has only 2 CPU and 7 GB of DRAM, we cannot use a big
+        # number here.
+        start args: ${{ matrix.driver == 'docker' && '-n 2' || '' }}
```

Comment:
I hope this isn't evaluating to `true` rather then '-n 2'. Did we try it already?